orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/DeploymentDetailsAware.groovy

```diff
@@ -34,7 +31,7 @@ import com.netflix.spinnaker.orca.pipeline.model.Stage
 trait DeploymentDetailsAware {
 
   private ObjectMapper pipelineObjectMapper =
-    new ObjectMapper().registerModule(new SimpleModule("MyConverterModule").addAbstractTypeMapping(Stage, PipelineStage))
+    new OrcaObjectMapper().registerModule(new SimpleModule("MyConverterModule").addAbstractTypeMapping(Stage, PipelineStage))
```

Comment:
Yes, because there's no type specifier in the stages as encoded in the trigger so it has to know to interpret them as PipelineStages.